app/controllers/application_controller.rb

```diff
@@ -219,6 +219,7 @@ def require_admin
   end
 
   def render_403
+    return head(:forbidden) unless respond_to?(:layout)
```

Comment:
This seems to break regular 403 pages for me, if I log in as a user and try to access a route that I don't have permission for.  What's `layout` meant to be in this context?